src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryController.kt

```diff
@@ -83,6 +83,19 @@ class LibraryController(private val libraryService: LibraryService) {
     }
   }
 
+  @PatchMapping("/{id}")
+  fun updateLibrary(
```

Comment:
tiny thing: can you change the name of this function to `updateLibraryName`? Just to make it super clear that this is only updating the name and not game data (or any other additional metadata we may need to add in the future).

Reply:
Whoops yeah good point!